hyrax.gemspec

```diff
@@ -67,7 +67,7 @@ EOF
   spec.add_dependency 'active_attr', '~> 0.9.0'
   spec.add_dependency 'redlock', '>= 0.1.2'
   spec.add_dependency 'retriable', '>= 2.9', '< 4.0'
-  spec.add_dependency 'active-fedora', '>= 11.1.3'
+  spec.add_dependency 'active-fedora', '~> 11.2'
```

Comment:
So now I'm wondering if there is something special about `11.1.3` that fixed critical bugs or added critical features.  In other words, I'm trying to understand what the most permissive constraint could be, like `>= 11.0.0` or say `~> 11.0`, and why the current constraint is not that permissive.